account_invoice_line_stock_move_info/README.rst

```diff
@@ -0,0 +1,10 @@
+account invoice line stock move info
+====================================
+This module informs to the invoice line, the movement of stock that
+corresponds.
+
+To use this module in the configuration of purchase and sale journals,
```

Comment:
For having this field correctly filled, you have to uncheck "Group Invoice Lines" option in the sale and purchase journals.